pipeline/authz/remote_json.go

```diff
@@ -42,9 +49,8 @@ type AuthorizerRemoteJSON struct {
 // NewAuthorizerRemoteJSON creates a new AuthorizerRemoteJSON.
 func NewAuthorizerRemoteJSON(c configuration.Provider) *AuthorizerRemoteJSON {
 	return &AuthorizerRemoteJSON{
-		c:      c,
-		client: httpx.NewResilientClientLatencyToleranceSmall(nil),
```

Comment:
Is it possible that this could cause a panic? I think we should leave the value initialized.

Reply:
Ok, i'm going to put back this line.
I had copied the behavior implemented in the Oauth2 authenticator.